frontend/public/src/components/EnrolledItemCard.js

```diff
@@ -54,17 +54,19 @@ type EnrolledItemCardProps = {
 type EnrolledItemCardState = {
   submittingEnrollmentId: number | null,
   activeMenuIds: number[],
-  emailSettingsModalVisibility: boolean[]
+  emailSettingsModalVisibility: boolean[],
+  verifiedUnenrollmentModalVisibility: boolean[]
 }
 
 export class EnrolledItemCard extends React.Component<
   EnrolledItemCardProps,
   EnrolledItemCardState
 > {
   state = {
-    submittingEnrollmentId:       null,
-    activeMenuIds:                [],
-    emailSettingsModalVisibility: []
+    submittingEnrollmentId:              null,
+    activeMenuIds:                       [],
+    emailSettingsModalVisibility:        [],
+    verifiedUnenrollmentModalVisibility: []
```

Comment:
The other two were here before, but I'm wondering why these are arrays when there is only ever one enrollment id this component sees? It'd make the logic in this component a lot simpler to just use an unwrapped boolean, defaulting to `false`. It'd probably cut the amount of new code you wrote here by more than half.

Reply:
Fixed this in https://github.com/mitodl/mitxonline/pull/602/commits/588fdaa73e4699676a4d9a71af791967427ccd65